src/server/mod.rs

```diff
@@ -425,8 +425,12 @@ impl LanguageServer for LspServer {
                     },
                 ),
                 text_document_sync: Some(
-                    lsp::TextDocumentSyncCapability::Kind(
-                        lsp::TextDocumentSyncKind::FULL,
+                    lsp::TextDocumentSyncCapability::Options(
+                        lsp::TextDocumentSyncOptions {
+                            open_close: Some(true),
+                            change: Some(lsp::TextDocumentSyncKind::FULL),
+                            ..Default::default()
```

Comment:
So the downstream defaults ([downstream of here](https://github.com/gluon-lang/lsp-types/blob/master/src/lib.rs#L1757)) all have the save options as default false. So we removed the save. 👍🏼